interface/patient_file/summary/add_edit_issue.php

```diff
@@ -840,5 +841,22 @@ function divclick(cb, divid) {
  Calendar.setup({inputField:"form_end", ifFormat:"%Y-%m-%d", button:"img_end"});
  Calendar.setup({inputField:"form_return", ifFormat:"%Y-%m-%d", button:"img_return"});
 </script>
+
+<!--Page Form Validations-->
+<?php require_once($GLOBALS['srcdir']."/validation/validation_script.js.php");?>
+<?php $collectThis=collectValidationPageRules("add_edit_issue",$_SERVER['PHP_SELF'])?>
```

Comment:
Thinking out loud. If you just use the path/filename in title to find it, then it would be really cool if could automate the form.id by also placing it somewhere inside the list_options structure. And in this case the option_id validator_id would not be used but we a good anchor if ever need to update the entry via the sql upgrade script. Note that if you embed the form.id somehow in there, then would then even have potential to send back more than 1 validation set if there is more than 1 form in the script.